frontend/translations/en.po

```diff
@@ -421,7 +421,7 @@ msgid "dashboard.download-binary-file"
 msgstr "Download Penpot file (.penpot)"
 
 msgid "dashboard.download-binary-file-v3"
```

Comment:
Missing spanish translation

Reply:
Yes, it already was missing. I have still pending tasks for it. This will be fixed in other PR